wails-frontend/src/main.tsx

```diff
@@ -21,12 +21,15 @@ const router = createBrowserRouter(
   createRoutesFromElements(
     <Route>
       <Route path="/" element={<App />} />
+      <Route path="/backup-methods" element={<BackupMethods />} />
+      <Route path="/backup-file" element={<BackupFile />} />
+      <Route path="/backup-pkey" element={<BackupKeyPairs />} />
```

Comment:
```suggestion
      <Route path="/backup-methods" element={<BackupMethods />} />
      <Route path="/backup-methods/file" element={<BackupFile />} />
      <Route path="/backup-methods/pkey" element={<BackupKeyPairs />} />
```

@mazmassa Correct me if it's not a good practice, but it is clearer if we indent rootes like this.

Reply:
we can't use slashes I think @Thykof

Reply:
In wails, we can use / in path. But there is not much interest to have very beautiful URLs for end user because end use will never see them.

If you want to add /

```html
<Route path="/backup-methods" element={<BackupMethods />} />
<Route>
  <Route path="/backup-methods/file" element={<BackupFile />} />
  <Route path="/backup-methods/pkey" element={<BackupKeyPairs />} />
</Route>
```